js/languages/en-US.json

```diff
@@ -1574,6 +1590,11 @@
     "totalPercentageTooLow": "The sum of the buyer and vendor amounts must add up to 100.",
     "provideResolution": "Please explain your decision."
   },
+  "searchProviderModelErrors": {
+    "name": "The name must be a string.",
```

Comment:
I don't think you want to use field names as the error keys since a field could have any number of potential errors. Those keys would be better as `provideName`, `invalidLogo`, `invalidListingUrl`. This would play nice if those fields had additional error, e.g. `nameTooLong`.